client/testclient/cmd/register.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"encoding/json"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+)
+
+var registerCmd = &cobra.Command{
+	Use:   "register",
+	Short: "Register a name-to-key binding.",
+	Long: `Register a new name-to-key binding on the CONIKS-server.
+Note that, for the registration to work, the test-client needs to be running on the same machine as the key-server.
+A real client would first register an app here: https://apps.twitter.com (the UI should make this possible without actually leaving the client application). Then, the client would use the oauth1 API (find the tokens on the [Keys and Access Tokens] tab of the aforementioned page) and contact the (twitter-)bot for a registration request. The twitter bot is also accessible from outside.
+For more information consult: https://godoc.org/github.com/coniks-sys/coniks-go/bots
+
+Example call:
+  coniksclient register -name Alice@twitter -key fake_test_key`,
```

Comment:
I should have sticked with the short version of flags in the usage string (which I tried myself) ;-) Thanks for testing it! Fixed in 	bd8d071